app/src/sharedTest/java/org/oppia/app/activity/InputInteractionViewTestActivityTest.kt

```diff
@@ -0,0 +1,88 @@
+package org.oppia.app.activity
+
+import android.app.Activity
+import android.content.Intent
+import android.content.pm.ActivityInfo
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import org.junit.Assert.*
```

Comment:
Prefer not wildcard importing; please import each symbol needed to avoid polluting the global symbol namespace in this file.

Reply:
Resolved